From 345884786b9b57424b6968472a97545cb42c53d0 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Sat, 27 Nov 2021 12:49:09 +0100
Subject: [PATCH] Add CMake option to build without X11

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 17 +++++++++++------
 1 file changed, 11 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 16e4884d..2a29338f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -132,13 +132,18 @@ endif()
 set_package_properties(KF5KDcraw PROPERTIES URL "https://invent.kde.org/graphics/libkdcraw/" DESCRIPTION "C++ interface around LibRaw library used to decode RAW picture files" TYPE OPTIONAL)
 
 
-find_package(X11)
-if(X11_FOUND)
-    if (QT_MAJOR_VERSION STREQUAL "5")
-        find_package(Qt5 ${QT_MIN_VERSION} CONFIG REQUIRED X11Extras)
+if(UNIX AND NOT APPLE)
+    option(WITH_X11 "Build with X11 integration" ON)
+    if(WITH_X11)
+        find_package(X11 REQUIRED)
+        if (QT_MAJOR_VERSION STREQUAL "5")
+            find_package(Qt5 ${QT_MIN_VERSION} CONFIG REQUIRED X11Extras)
+        endif()
+        # we need to add qt6 lib when we found x11
+        set(HAVE_X11 TRUE)
     endif()
-    # we need to add qt6 lib when we found x11
-    set(HAVE_X11 1)
+else()
+    set(WITH_X11 OFF)
 endif()
 
 find_package(kImageAnnotator)
-- 
2.35.1

