From 1c1a0fb839355ef3e684dac37189c09a7eafb5da Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Sat, 27 Nov 2021 13:52:23 +0100
Subject: [PATCH] Add CMake option to build without X11

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 091334066..22f5422b2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,10 +26,10 @@ include(FeatureSummary)
 find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS TextWidgets Notifications GuiAddons IconThemes WindowSystem KIO DBusAddons)
 find_package(Qt5 ${QT_MIN_VERSION} CONFIG REQUIRED COMPONENTS DBus)
 
-
-find_package(X11)
-if(X11_FOUND)
-   set(HAVE_X11 1)
+option(WITH_X11 "Build with X11 integration" ON)
+if (WITH_X11)
+	find_package(X11 REQUIRED)
+	set(HAVE_X11 1)
 endif()
 
 add_definitions(
-- 
2.34.1

